### ISSUE TITLE
Files to local - Failed. How to debug?

### ISSUE THREAD

**Issue by versant2612** [linkedpipes/etl#800]
I have two pipelines with Files to local component and it is the last component from the pipe. On both pipes these error message is shown. How to debug the Root Cause? The previous component is Tabular and runs successfully.

Component: Files to local
Status: Failed
Start: 2020-03-11 18:27:13
End: 2020-03-11 18:27:21
Duration: 00:00:08

Cause:
Can't copy runtime configuration.

Root cause:
IllegalArgumentException : Not a valid (absolute) IRI: node1e35k4uemx61214

Component: Files to local
Status: Failed
Start: 2020-03-11 18:29:48
End: 2020-03-11 18:29:49
Duration: 00:00:00

Cause:
Can't copy runtime configuration.

Root cause:
IllegalArgumentException : Not a valid (absolute) IRI: node1e35k4uemx119726

**Comment by jakubklimek** [comment]
@versant2612 Without a screenshot of the pipeline or the pipeline itself, I am only guessing here.
You have probably connected a component with RDF output to the Runtime configuration input of Files to local (the pink one). That is intended for advanced use cases when you actually want to generate a configuration for the component, which is not the case here.
What you probably need to do is connect the RDF output of your component to [RDF to File](https://etl.linkedpipes.com/components/t-rdftofile) (dumping RDF data to a file). Then, connect the output of RDF to File to [Files to local](https://etl.linkedpipes.com/components/l-filestolocal) file input (the green one).

**Comment by versant2612** [comment]
Yes, your guess is right. I did what you suggested (used RDF for file) and it worked. The entire pipeline was executed and the result file was stored in the local directory.
Analyzing the output file, I identified that I need to convert some predicates into a common vocabulary so I inserted a SPARQL Update component after Tabular and now I have the same error.

![image](https://user-images.githubusercontent.com/46487949/76711308-9289ad00-66ed-11ea-8347-a51582cb1486.png)
